### PR TITLE
Add atl08

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**
Tested via deployment to https://or569ue0p9.execute-api.us-east-1.amazonaws.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,29 +14,29 @@ First, add your AWS credentials to a new file called `.env`. You can see an exam
...
 
-**IMPORTANT:** Create if needed and ensure access to the buckets configured in `stack/config.yml`.
+Note, the local `stack/config.yml` file will only be used for running the app locally. Deployment to AWS is managed via CDK and github actions (See `.github/workflows/deploy.yml`).
+
+Datasets for `/v1/datasets` are loaded from a json file stored in S3 unless `ENV=local` is set when running the app. The S3 location for these datasets is defined by the `BUCKET` and `DATASET_METADATA_FILENAME` values in `stack/config.yml`: `s3://{BUCKET}/{DATASET_METADATA_FILENAME}`.
 
 ### Running the app locally
 
-To run the app locally, generate a config file and generate the static dataset json files.
+You can use `ENV=local` when running the app locally to use the `example-dataset-metadata.json` file as the source for `/v1/datasets`. This is useful for testing new dataset configurations.
```
Comment (slesaad) [inline README.md:27]: Do we have this set up?

Comment (abarciauskas-bgse) [reply README.md:27]: Yes? I've added this conditional for local file reading - https://github.com/MAAP-Project/dashboard-api-maap/blob/ba75b0406bfe175bc043deed2ca746229bd02c5c/dashboard_api/db/static/datasets/__init__.py#L36-L40

Comment (slesaad) [reply README.md:27]: ahh, sorry i didn't see that

**Review by slesaad — APPROVED** [review]
LGTM

**Pull request merged**